product_end_of_life/models/product.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# Integrators License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import datetime, date
+from dateutil.relativedelta import relativedelta
+from odoo import api, fields as fields_cls, models
```

Comment:
I see no reason to import `fields` as `fields_cls`

Reply:
Method on line https://github.com/OCA/product-attribute/pull/328/files#diff-cabc9687719397e1da647f8dc3ab2b25R92 has a named attribute ```fields``` which is used in the said method